packages/grid/_modules_/grid/hooks/features/pagination/usePagination.ts

```diff
@@ -102,12 +102,6 @@ export const usePagination = (apiRef: ApiRef): void => {
     dispatch(setRowCountActionCreator({ totalRowCount: visibleRowCount }));
   }, [apiRef, dispatch, visibleRowCount]);
 
-  React.useEffect(() => {
```

Comment:
I tried to quickly fix it by doing that but if I remember correctly removing this messes up the initial render

Reply:
Any idea where I can see the issue?

Reply:
Found it :)